src/SFA.DAS.EmployerIncentives.Api/Controllers/PassThroughController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Interfaces;
+using SFA.DAS.EmployerIncentives.Models.PassThrough;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [ApiController]
+    public class PassThroughController : ControllerBase
```

Comment:
I'm not convinced on the name as this will get massive!  I'd rather preserve the same controller structure as we have in the inner API.

Reply:
The intent of this controller is to simply pass forward the request to the inner api, where we do not need to do anything to it.
It would be worth splitting this into queries requests and non-query requests. As polly can be applied to the request queries.

The name is worthwhile discussing but the intent of the code here is to quickly forward it to the inner API and return the response.